libcfnet/file_stream.c

```diff
@@ -829,6 +836,20 @@ static bool SendSignature(SSL *conn, const char *filename)
     fclose(file);
     rs_job_free(job);
 
+    const char *msg =
+        "Send signature statistics:\n"
+        "  %zu bytes in (read from '%s')\n"
+        "  %zu bytes out (sent to server)\n";
+
+    if (print_stats)
+    {
+        printf(msg, bytes_in, filename, bytes_out);
```

Comment:
I'd personally do `fprintf(stderr, ...)`, but we generally don't use the error output so this is probably fine.